src/components/responsive-flyout/styles/OverlayHeader.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../styles/variables';
+@import './variables';
+
+.overlay-header {
+    position: relative;
+    z-index: 1;
```

Comment:
We have to be careful with z-index values, since they can often conflict. Is this needed?

Reply:
I wanted to make sure OverlayHeader is above the content since it contains a drop shadow